doc/self-hosting.md

```diff
@@ -93,6 +93,23 @@ machine.
 
 [shields.example.env]: ../shields.example.env
 
+## Raster server
+
+If you want to host PNG badges, you can also self-host a [raster server][]
+which points to your badge server.
+
+- In your raster instance, set `BASE_URL` to your Shields instance, e.g.
+  `https://shields.example.co`.
+- Optionally, in your Shields, instance, configure `RASTER_URL` to the base
+  URL, e.g. `https://raster.example.co`. This will send 301 redirects
+  for the legacy raster URLs instead of 404's.
+
+If anyone has set this up, more documentation on how to do this would be
+welcome! It would also be nice to ship a Docker image that includes a
```

Comment:
Do you think its worth adding a sentence just saying its designed to run as a lambda on now/AWS/etc, or is that stuff just best covered in the README for `svg-to-image-proxy`?

Reply:
I think that's a fine thing to add. If it's information that isn't going to change, it's not the worst thing to mention it in two places.